### PR TITLE
feat: switch to simpler serge/localization set-up

### PR DESCRIPTION
**Pull request by devpow112**
Switch to simpler set-up where locales doesn't need to be built and we use the dynamic localization mixin. This required updating eslint as it couldn't figure out the dynamic localize mixin stuff.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "brightspace/lit-config"
+  "extends": "brightspace/lit-config",
+  "rules": {
+    "sort-class-members/sort-class-members": 0
```
Comment (devpow112) [inline .eslintrc.json:4]: Ignore for now as it makes way too much noise. This PR is already large enough.

File: add-associations-launcher.js [hunk 2]
```diff
@@ -21,19 +19,6 @@ class AddAssociationsLauncher extends LocalizeMixin(LitElement) {
 		return css``;
 	}
 
-	static async getLocalizeResources(langs) {
```
Comment (devpow112) [inline add-associations-launcher.js:24]: Removed in favour of dynamic localization mixin

File: add-associations.js [hunk 3]
```diff
@@ -90,19 +89,6 @@ class AssociationList extends LocalizeMixin(LitElement) {
 		`;
 	}
 
-	static async getLocalizeResources(langs) {
```
Comment (devpow112) [inline add-associations.js:93]: Removed in favour of dynamic localization mixin

File: add-associations-launcher.js (hunk 2 above)
Comment (mpharoah-d2l) [reply add-associations-launcher.js:24]: I don't see anything that was calling this. Was it even used before?

**Pull request merged**